src/content/docs/en/recipes/docker.mdx

```diff
@@ -148,10 +148,10 @@ WORKDIR /app
 COPY package.json package-lock.json ./
 
 FROM base AS prod-deps
-RUN npm install --production
+RUN npm install --omit=dev
 
 FROM base AS build-deps
-RUN npm install --production=false
+RUN npm install
 
 FROM build-deps AS build
 COPY . .
```

Comment:
Niptick: While it works, I would instead specify the intended folders to copy. This solution heavily relies on `.dockerignore` being correct and not allowing anything unwanted to be copied.

Reply:
Suggestion/Nitpick: While `.` works fine, perhaps `/app/` would convey more information without relying on the previous state of the `WORKDIR`.

Reply:
it's different working styles, I guess the doc purpose is only to convey examples, it's hard to cover all use cases, and sometimes people use them in different context, some even develop with live reload under docker. relying on workdir also prevents changing it everywhere in case of rename.

Reply:
I would still advise against using `COPY . .` to avoid beginner mistakes. With HRM you would mount the files, and that is fine, but when you `COPY`, you do copy files into your image! You can very easily make a mistake and blow up your image size and even leak sensitive information with things like `.env`.

Reply:
FYI: https://discord.com/channels/830184174198718474/1245118392793436201